geopandas/explore.py

```diff
@@ -225,6 +226,10 @@ def my_colormap(value):  # scalar value defined in 'column'
             Applies if ``colorbar=False``.
         max_labels : int, default 10
             Maximum number of colorbar tick labels (requires branca>=0.5.0)
+    map_kwds : dict (default {})
+        Additional keywords to be passed to :class:`folium.Map`,
```

Comment:
```suggestion
        Additional keywords to be passed to folium :class:`~folium.folium.Map`,
```
Folium doesn't expose `Map` in the main namespace so the link is not applied. This does the trick.